cpp/ps0003/p3Solution.cpp

```diff
@@ -9,11 +9,13 @@ void checkEvenorOdd(int num) {
 }
 
 int main() {
+	// prompting user for number 2/2
 	int num;
 	cout << "Enter a number to check if it's even or odd: ";
 	cin >> num;
-	checkEvenorOdd(num);
 
+	// checking if the number is odd or even 3/3
+	checkEvenorOdd(num);
 	return 0;
-
-}
\ No newline at end of file
+}
+// great work 😎 10/10 ⭐

```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
+1 bonus mark for using functions